.test-defs/cmd/shoot-maintenance/main.go

```diff
@@ -57,11 +60,87 @@ var (
 	testMachineImage = gardenv1beta1.ShootMachineImage{
 		Version: testMachineImageVersion,
 	}
+
+	shootMaintenanceTest          *framework.ShootMaintenanceTest
+	gardenerSchedulerReplicaCount *int32
 )
 
 func init() {
+	ctx := context.Background()
+	defer ctx.Done()
 	testLogger = logger.NewLogger("debug")
 
+	setEnvironmentVariables()
```

Comment:
```suggestion
	setEnvironmentVariables(ctx)
```

Reply:
why is there the ctx needed? The function itself never uses any context.